website_product_supplier/views/product_supplier_view.xml

```diff
@@ -0,0 +1,204 @@
+                <div class="col-md-12 col-xs-12">
+                    <!-- Show error information -->
+                    <div class="alert alert-warning" t-if="error">
+                        <i class="fa fa-exclamation-triangle"/><span t-esc="error.get('error_name', '')"/>
+                    </div>
+                    <form id="supplier_product"
+                          t-attf-action="/my/supplier/product/save/#{main_obj and slug(main_obj) or ''}"
+                          method="post" enctype="multipart/form-data">
+                        <div class="panel panel-default">
+                            <div class="panel-heading">
+                                <h3 class="panel-title"><span t-esc="supplierinfo.get('product_name','New product')"/></h3>
+                            </div>
+                            <div class="panel-body">
+                                <div class="row">
+                                    <div id="product_image" class="col-md-4">
+                                        <span t-field="product.image"
+                                              t-field-options='{"widget": "image", "class": "product_detail_img", "alt-field": "name"}'
+                                              class="oe_form_field_image_controls"/>
+                                        <div class="text-center">
+                                            <label class="custom-file-input">
+                                                <div class="btn btn-default">Load Picture</div>
+                                                <input class="input-file" id="fileInput" type="file" name="ufile"/>
+                                            </label>
+                                        </div>
+                                    </div>
+                                    <div id="product_fields" class="col-md-8">
+                                        <section id="product_fields_base">
+                                            <div id="product_name" class="row">
+                                                <div t-attf-class="col-md-8 col-xs-12 form-group #{error.get('product_name') and 'has-error' or ''}">
+                                                    <label class="control-label">Name</label>
+                                                        <input id="product_name" class="form-control" name="product_name" placeholder="Product name..."
+                                                               t-att-value="supplierinfo.get('product_name')"/>
+                                                </div>
+                                                <div t-attf-class="col-md-4 col-xs-12 form-group #{error.get('product_code') and 'has-error' or ''}">
+                                                    <label class="control-label">Code</label>
+                                                        <input id="product_code" class="form-control" name="product_code" placeholder="Product Code..."
```

Comment:
Placeholders should indicate examples, not repeat the field label.

For example, this field is labeled "Code", and the placeholder could be something like "123456".

The same applies for all other fields.